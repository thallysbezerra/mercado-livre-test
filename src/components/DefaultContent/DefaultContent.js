import React from 'react';
import './DefaultContent.scss';

const DefaultContent = () => (
	<div className="default-content">Busque un producto en campo arriba.</div>
);

export default DefaultContent;
