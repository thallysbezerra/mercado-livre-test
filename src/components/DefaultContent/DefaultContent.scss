@import '../../css/theme';

.default-content {
	font-size: 28px;
	font-weight: $medium;
	margin: 150px auto 0;
	position: relative;
	transform: translateY(-50%);
	width: 300px;

	&:after {
		background-image: url('./img/arrow-up.png');
		content: '';
		height: 100px;
		position: absolute;
		top: -110px;
		width: 65px;

		@include breakpoint(tabletL) {
			right: -75px;
			top: -100px;
		}
	}
}
