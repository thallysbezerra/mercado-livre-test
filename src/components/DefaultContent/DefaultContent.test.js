import React from 'react';
import DefaultContent from './DefaultContent';

import renderer from 'react-test-renderer';

describe('DefaultContent UI', () => {
	it('<DefaultContent /> must have the snapshot version.', () => {
		const tree = renderer.create(<DefaultContent />);

		expect(tree.toJSON()).toMatchSnapshot();
	});
});
