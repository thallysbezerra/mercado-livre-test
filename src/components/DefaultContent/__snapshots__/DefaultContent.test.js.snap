// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultContent UI <DefaultContent /> must have the snapshot version. 1`] = `
<div
  className="default-content"
>
  Busque un producto en campo arriba.
</div>
`;
